hbase_storage/src/main/java/com/splicemachine/hbase/TransactionsWatcher.java

```diff
@@ -34,17 +34,23 @@
 public class TransactionsWatcher {
     private static final Logger LOG = Logger.getLogger(TransactionsWatcher.class);
     private static final AtomicBoolean started = new AtomicBoolean(false);
-    private static AtomicLong oldestActiveTransaction = new AtomicLong(Long.MAX_VALUE);
+    private static AtomicLong lowWatermarkTransaction = new AtomicLong(Long.MAX_VALUE);
 
     public static TransactionsWatcher INSTANCE = new TransactionsWatcher();
 
     private static ScheduledExecutorService updateService =
             MoreExecutors.namedSingleThreadScheduledExecutor("hbase-transactions-watcher-%d");
 
     private static final Runnable updater = () -> {
-        oldestActiveTransaction.set(fetchOldestActiveTransaction());
+        long fetchTimestamp = SIDriver.driver().getTimestampSource().nextTimestamp();
```

Comment:
This will increase timestamp. Does retriveTimestamp work?

Reply:
I'm not quite sure that retrieveTimestamp works here, especially since it does not seem to using an atomic operation underneath. Unless this is blocking, I would address that in a separate change.